opensrp-chw-referral/src/test/java/org/smartregister/chw/referral/custom_views/BaseReferralFloatingMenuTest.java

```diff
@@ -6,20 +6,22 @@
 import org.mockito.Mockito;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.rule.PowerMockRule;
+
 @PrepareForTest(BaseReferralFloatingMenu.class)
 public class BaseReferralFloatingMenuTest {
```

Comment:
Could also test the core functionality of this class? What has been added so far is good but to avoid regressions, the reported test coverage should reflect the actual unit test of the functionality and the edge cases.